deer/demo.py

```diff
@@ -124,7 +124,7 @@ def main():
     print(f"DEER GRU speed up over sequential GRU: {seq_time / deer_time:.3f}x")
 
     # calculate the error
-    dev = jnp.abs(outputs1 - outputs2).max()
+    dev = jnp.abs(outputs1 - outputs2.value).max()
```

Comment:
It's better to get the value inside the `eval_gru` function, for consistency. Could you please do that?

Reply:
Ok, I will make the change soon